static/index.html

```diff
@@ -11,6 +11,10 @@
         document.getElementById('antiClickjack').innerHTML =
           '#content__framed-body { display: none !important; }';
       }
+
+      window.onload = function() {
+        document.getElementById("suspect-link").innerHTML = document.referrer;
```

Comment:
Setting `.innerHTML` can have [security implications](https://dev.to/caffiendkitten/innerhtml-cross-site-scripting-agc).  In this case innerText should work as well 👍 

```suggestion
        document.getElementById("suspect-link").innerText = document.referrer;
```